src/utils/OpenAI/OpenAI.ts

```diff
@@ -53,6 +53,11 @@ export const getOpenAICompletion = async (
 
           try {
             const json = JSON.parse(data);
+            const finishReason = json.choices[0].finish_reason || "";
+            if (finishReason === "stop") {
```

Comment:
adhoc fixes for now, since the termination data is not compatible with that of OpenAI's.